RecipesLyricsMashup/RecipesLyricsMashupV2/RecipeLyricsMashup/RecipeLyricsMashup/Pages/Index.cshtml.cs

```diff
@@ -19,23 +19,26 @@ public IndexModel(ILogger<IndexModel> logger)
         {
             _logger = logger;
         }
+        [BindProperty(SupportsGet = true)]
+        public string SearchString { get; set; } //Morgan added

```

Comment:
Added this variable as a way to pass the last searched string to and from the front-end page.